src/lib/NodeHandle.js

```diff
@@ -195,11 +195,11 @@ class NodeHandle {
 
     // don't namespace action client - topics will be resolved by
     // advertising through this NodeHandle
-    return new ActionClient({
+    return new ActionClient(Object.assign(options, {
```

Comment:
Rather than editing the user's referenced `options` object, should use a dummy empty object as the target:
`Object.assign({}, options, {actionServer, ...})`